projects/angular/src/button/_buttons.clarity.scss

```diff
@@ -122,6 +121,11 @@
     } @else {
       $fallbackColor: getSassButtonColor($button-type, box-shadow-color);
 
+      // ATM (after token migration)
+      // Replace box-shadow section with commented section. Renaming all of clr-btn 'box-shadow' to 'click' color properties is needed.
+      // $buttonColor: 'clr-btn-' + $button-type + '-click-color';
+      // @include mixins.css-var(background-color, $buttonColor, $fallbackColor, variables.$clr-use-custom-properties);
+
       // IE/OldEdge
```

Comment:
ATM we may want to remove all these, if they are only IE compatibility related.

Reply:
I'll add those to the JIRA tiket.